tekton-catalog/pipeline-loops/pkg/reconciler/pipelinelooprun/controller.go

```diff
@@ -128,13 +128,13 @@ func initLogger(ctx context.Context, kubeClientSet kubernetes.Interface) *zap.Su
 		MaxSize: 1024 * 100, // TODO make it configurable via a configmap.
 	}
 	enabled, err := loadObjectStoreConfig(ctx, kubeClientSet, &loggerConfig)
-	if err == nil {
+	if err == nil && enabled {
 		err = objectStoreLogger.LoadDefaults(loggerConfig)
 		if err == nil {
 			_ = objectStoreLogger.Writer.CreateNewBucket(loggerConfig.DefaultBucketName)
+		} else {
+			logger.Errorf("error connecting to the object store, %v", err)
 		}
-	}
-	if err == nil && enabled {
 		logger.Info("Loading object store logger...")
```

Comment:
my apology. when I mentioned merging the condition checks, I didn't mean just combining them together. I know from here below you still need to check if the `objectStoreLogger` is properly loaded or not, right? I don't know the code below can run properly without a valid objectStroeLogger.